javascript/src/com/google/idea/blaze/javascript/run/producers/NonBlazeProducerSuppressor.java

```diff
@@ -40,6 +39,9 @@ public class NonBlazeProducerSuppressor implements StartupActivity {
       PRODUCERS_TO_SUPPRESS =
           ImmutableList.of(
               JestRunConfigurationProducer.class,
+              // Not suppressing JestRunConfigurationProduce since that prevents Jest tests from
+              // running properly, see https://github.com/bazelbuild/intellij/issues/3629
+              // JestRunConfigurationProducer.class,
```

Comment:
I think you forgot to remove it here:
```suggestion
              // Not suppressing JestRunConfigurationProduce since that prevents Jest tests from
              // running properly, see https://github.com/bazelbuild/intellij/issues/3629
              // JestRunConfigurationProducer.class,
```

Reply:
Oops, well spotted, thanks Uri 👍 you can tell I lazily made this commit using github editor, apologise 😂

Confirm no other changes
<img width="1430" alt="image" src="https://user-images.githubusercontent.com/2678063/180670083-68415add-5b4e-4d5e-8168-b34bce6f7608.png">